src/tests/mod.rs

```diff
@@ -184,35 +164,42 @@ impl Tester {
             }
         }
     }
-    fn static_run(&self, cfg: &Config, sol: PVTSolution) {
-        let reference = self.reference.as_ref().unwrap();
-        let xyz_ecef_m = reference.ecef();
-        let (x0, y0, z0) = (xyz_ecef_m[0], xyz_ecef_m[1], xyz_ecef_m[2]);
-        let (x, y, z) = (sol.position[0], sol.position[1], sol.position[2]);
-        let (vel_x, vel_y, vel_z) = (sol.velocity[0], sol.velocity[1], sol.velocity[2]);
-        let (x_err, y_err, z_err) = ((x - x0).abs(), (y - y0).abs(), (z - z0).abs());
+    fn static_run(&self, _cfg: &Config, sol: PVTSolution) {
+        //let reference = self.reference.as_ref().unwrap();
```

Comment:
Is it expected to comment out these tests? I don't know your library well, but I wouldn't want the switch to ANISE to hide new bugs.

Reply:
👍 
this test does not do anything to this day, it's still a sketch to try to put the solver to the test.
I have another pending PR to try to introduce some ecosystem to test it, but I'm not sure I will conclude it either. 
The other PR is kind of thorough and is close to generate something.. but it's also the exact same purpose of my rinex-cli application. It's kind of annoying writing the same kind of thing twice. But it's most certainly not sustainable if we don't have continuos tests of the solver to track how it evolves. It is complex to set up, and is a complete topic on its own.
It took months to get rinex-cli to interface nicely and answer most of the requirements. Even to this day, some are still impossible, some combinations are not feasible or not properly supported.